EnglishFilter/sections/foreign.txt

```diff
@@ -9165,6 +9165,13 @@ brandbuffet.in.th#@#.category-advertorial
 !------------------------------ Vietnam ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76063
+coinurl.net#%#//scriptlet("set-constant", "adsHeight", "15")
+coinurl.net#%#//scriptlet("prevent-window-open")
+coinurl.net##div > [href]
+coinurl.net##a[href^="https://vnbit.org/"]
+!+ PLATFORM(ios, ext_safari, ext_android_cb)
+coinurl.net#@#[id*="ScriptRoot"]
```

Comment:
I guess `coinurl.net#%#//scriptlet("set-constant", "adsHeight", "15")` is not required with this exclusion.

Reply:
Required that only for ios, safari and cb. There is a leftover if I use it for adguard for windows.